### PR TITLE
Add peruvianidol theme

### PR DESCRIPTION
**Pull request by peruvianidol**
Adds a new theme based on twitch.tv/peruvianidol

### CODE REVIEW

File: src/themes/peruvianidol.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+[data-twitch-sender-roles='mod'] .twitch-chat-sender,
+[data-twitch-sender-roles='mod subscriber'] .twitch-chat-sender {
+	color: var(--color-mod);
+}
+
+[data-twitch-sender-roles='mod'] .twitch-chat-sender:before {
+	content: '👮🏻‍♂️';
+}
+
+[data-twitch-sender-roles='mod subscriber'] .twitch-chat-sender:before {
```
Comment (BenDMyers) [inline src/themes/peruvianidol.css:138]: I totally just asked to consolidate a line above, but I think this line could benefit from splitting out the mod and subscriber checks:

```suggestion
[data-twitch-sender-roles~='mod'][data-twitch-sender-roles~='subscriber'] .twitch-chat-sender:before {
```

This way, we'll also match accounts that are mods, subscribers, and VIPs.

File: src/themes/peruvianidol.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+[data-twitch-sender-roles='founder'] .twitch-chat-sender:before {
+	content: '👑';
+}
+
+[data-twitch-sender-roles='subscriber'] .twitch-chat-sender:before {
+	content: '⭐';
+}
+
+[data-twitch-sender-roles='mod'] .twitch-chat-sender,
+[data-twitch-sender-roles='mod subscriber'] .twitch-chat-sender {
```
Comment (BenDMyers) [inline src/themes/peruvianidol.css:130]: I think we might be able to cut this down a bit (and save on headaches when some folks could be mods, subscribers, _and_ VIPs) by going with the `~=` operator:

```suggestion
[data-twitch-sender-roles~='mod'] .twitch-chat-sender {
```

**Review by BenDMyers — CHANGES_REQUESTED** [review]

**Comment by BenDMyers** [comment]
The homepage select option formatting is turning the name `peruvianidol.css` and displaying it as `Peruvianidol`. Maybe we could rename this `peruvian-idol.css` so it displays as `Peruvian Idol`?

File: src/themes/peruvianidol.css (hunk 1 above)
Comment (peruvianidol) [reply src/themes/peruvianidol.css:138]: Ah, yeah, I like that better!

**Comment by BenDMyers** [comment]
@all-contributors Please add @peruvianidol for design

**Review by BenDMyers — APPROVED** [review]

**Pull request merged**